src/Microsoft.DotNet.ProjectJsonMigration/ProjectMigrator.cs

```diff
@@ -99,6 +97,11 @@ private void VerifyProject(IEnumerable<ProjectContext> projectContexts, string p
             }
         }
 
+        private string FormatDiagnosticMessage(DiagnosticMessage d)
```

Comment:
DiagnosticMessage already has a `FormattedMessage` property.  Why are we not using it?

---
In reply to: [79760911](https://github.com/dotnet/cli/pull/4216#discussion_r79760911) [](ancestors = 79760911)